includes/class-dispatcher.php

```diff
@@ -96,9 +96,9 @@ public static function process_outbox( $id ) {
 	 * @return array|void The next batch of followers to process, or void if done.
 	 */
 	public static function send_to_followers( $outbox_item_id, $batch_size = ACTIVITYPUB_OUTBOX_PROCESSING_BATCH_SIZE, $offset = 0 ) {
-		$json    = Outbox::get_activity( $outbox_item_id )->to_json();
-		$actor   = Outbox::get_actor( \get_post( $outbox_item_id ) );
-		$inboxes = Followers::get_inboxes_for_activity( $json, $actor->get__id(), $batch_size, $offset );
+		$outbox_item = \get_post( $outbox_item_id );
+		$json        = Outbox::get_activity( $outbox_item_id )->to_json();
```

Comment:
```suggestion
		$json        = $outbox_item->post_content;
```
Would this work, too?

Reply:
Oh nice! Will check next week!